docs/specs/xref.yml

```diff
@@ -338,6 +350,20 @@ repos:
             "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
             "xref": "a"
           }
+        _themes/ContentTemplate/schemas/TestData.schema.json: |
+          {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "uid": { "contentType": "uid" },
+              "xref": { "contentType": "Xref" },
+              "name": { "type": "string" },
+              "fullName": { "type": "string" },
+              "description": { "type": "string" },
+              "$schema": { "type": "string" }
```

Comment:
```suggestion
              ~~~"$schema": { "type": "string" }~~~
```

Reply:
This change will cause test failure, why we need this change?

Reply:
Oops, I meant removing these unnecessary lines like ~"$schema": { "type": "string" }~ and ~"type": "object",~ and ~"additionalProperties": false,~

Reply:
Updated. Sorry for missing this one.